pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -47,15 +58,14 @@ Panels::GSWindowSettingsPanel::GSWindowSettingsPanel( wxWindow* parent )
 	m_check_HideMouse	= new pxCheckBox( this, _("Always hide mouse cursor") );
 	m_check_CloseGS		= new pxCheckBox( this, _("Hide window when paused") );
 	m_check_Fullscreen	= new pxCheckBox( this, _("Default to fullscreen mode on open") );
-	m_check_VsyncEnable	= new pxCheckBox( this, _("Wait for Vsync on refresh") );
 	m_check_DclickFullscreen = new pxCheckBox( this, _("Double-click toggles fullscreen mode") );
 	m_check_AspectRatioSwitch = new pxCheckBox(this, _("Switch to 4:3 aspect ratio when an FMV plays"));
 	//m_check_ExclusiveFS = new pxCheckBox( this, _("Use exclusive fullscreen mode (if available)") );
 
 	m_text_Zoom->SetToolTip( pxEt( L"Zoom = 100: Fit the entire image to the window without any cropping.\nAbove/Below 100: Zoom In/Out\n0: Automatic-Zoom-In untill the black-bars are gone (Aspect ratio is kept, some of the image goes out of screen).\n  NOTE: Some games draw their own black-bars, which will not be removed with '0'.\n\nKeyboard: CTRL + NUMPAD-PLUS: Zoom-In, CTRL + NUMPAD-MINUS: Zoom-Out, CTRL + NUMPAD-*: Toggle 100/0"
 	) );
 
-	m_check_VsyncEnable->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
+	m_combo_vsync->SetToolTip( pxEt( L"Vsync eliminates screen tearing but typically has a big performance hit. It usually only applies to fullscreen mode, and may not work with all GS plugins."
```

Comment:
> I need to add code to check the proper driver support.

I see you are always a step ahead :*

> Either you drop a frame (GL) or your stall the GPU (DX)

I believe that \*especially\* for our use-case, GL behavior is actually golden. 

> You mean size of the window different of size of monitor. 

Yep. 

> I don't know how to query the size of the monitor. Besides what will happen in the next driver upgrade. 

Afaiu, for as much all of these are drivers quirks, they are not there at random. 
AFAIK, in Windows there's no way to set opengl (or I guess anything but DX) to fullscreen (or perhaps it was that *the application* has no way to know when it's what), so this is why they do this. 
Which I guess now with variable refresh rate monitors is more important then ever. 
I'm pretty confident this is expected and desired behavior on their side. 

To check, I had written [here](https://github.com/PCSX2/pcsx2/issues/1437#issuecomment-300828554). Though really, I understand if you prefer not to think about this. 

> What must happen for a 4:3 surface on a 16:9 monitor.

Who cares? Stretch/fit/fill/keep user preference is what controls this affair.